airflow/ui/src/pages/Run/TaskInstances.tsx

```diff
@@ -86,12 +105,72 @@ const columns: Array<ColumnDef<TaskInstanceResponse>> = [
   },
 ];
 
+const stateOptions = createListCollection({
+  items: [
+    { label: "All States", value: "all" },
+    { label: "Scheduled", value: "scheduled" },
+    { label: "Queued", value: "queued" },
+    { label: "Running", value: "running" },
+    { label: "Success", value: "success" },
+    { label: "Restarting", value: "restarting" },
+    { label: "Failed", value: "failed" },
+    { label: "Up For Retry", value: "up_for_retry" },
+    { label: "Up For Reschedule", value: "up_for_reschedule" },
+    { label: "Upstream failed", value: "upstream_failed" },
+    { label: "Skipped", value: "skipped" },
+    { label: "Deferred", value: "deferred" },
+    { label: "Removed", value: "removed" },
+  ],
```

Comment:
This list feels like a redundant definition of... what we have in Python in the backend. Is there a way to ensure this list is consistent with the backend?

Reply:
I couldn't find a mapping like this anywhere and the run tab in dagrun also had this manual entry.  `TaskInstanceState` from `openapi-gen/requests/schema.gen.ts` could be used but would still need transforming it to `label` field like underscore to space and captialization.

Reply:
Yeah, I also feel like (besides adding this to REST API) there is no common way to define cross language constant definitions like this, maybe we need to live with the redundancy until somebody knows a better way to share these common definitions...